### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR on Default Vercel Domains, Fine on Custom Domains - SSL Issue on PC Access

### DISCUSSION THREAD

**Discussion by plastikmind** [vercel/community#5111] [Help]
### Summary

Dear Vercel Support Team,

I'm experiencing an "ERR_SSL_PROTOCOL_ERROR" when accessing my Vercel-hosted sites on default domains via PC, but not on mobile. Interestingly, sites with custom domains don't have this issue. This problem persists across different PCs and browsers, and occurs even when using phone internet on a PC. 

It seems there might be a specific issue with SSL configurations for Vercel's default domains. Could you assist in resolving this discrepancy between default and custom domains? Any help would be greatly appreciated.

Best regards

### Steps to Reproduce

see some examples. all these sites work on vercel. It seems there is a specific issue with SSL configurations for Vercel's default domains.

https://sozluk-ilad.vercel.app
https://www.studiomuz.com
https://moon-message.vercel.app
https://www.koksalatay.com

**Comment by amyegan** [comment]
Thanks for reporting this. We had a recent TLS incident that is resolved now: https://www.vercel-status.com/incidents/sfrbn1scg6wn

Please let me know if you continue to have trouble with your sites.

**Reply by concman** [reply]
I also have the same issue with the secure site connection, subdomains of vercel are broken. Can't open the preview link that has been deployed by vercel.

**Comment by mehmetyildizdev** [comment]
I also have the same issue as the post owner. Custom domains are working fine but ...vercel.app seems to give that SSL error in any browser.

**Comment by amyegan** [comment]
These issues may be related to an ISP level block. We reached out to the provider to remove the block. Please follow in the main thread for the latest updates: https://github.com/vercel/community/discussions/5102
